ObjectOrientedProgramming/Memory Ownership/New and delete operators/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+cmake_minimum_required(VERSION 3.26)
```

Comment:
Please, don't use spaces in directory and file names. 
This can bring us unwanted problems on some platforms (e.g. Windows).
For example, instead of `Memory Ownership` a proper name should be `MemoryOwnership`. 
If you want a different name to be displayed (e.g. as a task name in "task desription" window in IDE), you can set in `.yml` file using `custom_name` property (you can find examples in this course).